test/sql/plan_ordered_append_analyze.sql

```diff
@@ -0,0 +1,9 @@
+-- This file and its contents are licensed under the Apache License 2.0.
```

Comment:
please put a comment as to why running analyze on this is important. Also why are we running both explain and explain analyze tests. Shouldn't the latter have strictly more info?

Reply:
you cant turn the timing info off on 9.6 thats why there is a version without analyze

Reply:
added comment